### DISCUSSION TITLE
简悦已经支持了大多数的导出服务，如果没有支持你常见的服务，可以看这里

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1557] [使用技巧]
> 如果简悦没有包含你需要的服务，请通过 **[投票方式](https://wj.qq.com/s2/8043542/c6bd)** 来选出你希望集成的服务，简悦会在下个版本中集成一些 **合理且投票数量最多** 的服务。

***

问题如上，

### 解决办法

> 利用 `临时阅读模式` 解决。

### 步骤：

- 进入阅读模式；
- 鼠标右下角控制栏；
- 动作 → 保存 → （点击）最后一个icon 

![Xnip2019-04-08_10-57-37](https://user-images.githubusercontent.com/81074/55695897-39010480-59ed-11e9-8a2a-2abe6df757c3.jpg)

**重要** 

> （稍等片刻）会生成一个阅读模式的临时页面，在这个页面如同你经常使用的任何页面一样，使用简悦暂时无法支持的服务。

**Comment by kid941005** [comment]
思源笔记

**Reply by Kenshin** [reply]
早就有了 😂  https://github.com/Kenshin/simpread/discussions/5592
